airflow/gcp/operators/dataflow.py

```diff
@@ -167,18 +168,18 @@ class DataFlowJavaOperator(BaseOperator):
     @apply_defaults
     def __init__(
             self,
-            jar,
-            job_name='{{task.task_id}}',
-            dataflow_default_options=None,
-            options=None,
-            gcp_conn_id='google_cloud_default',
-            delegate_to=None,
-            poll_sleep=10,
-            job_class=None,
-            check_if_running=CheckJobRunning.WaitForRun,
-            multiple_jobs=None,
+            jar: str,
+            job_name: str = '{{task.task_id}}',
+            dataflow_default_options: Optional[dict] = None,
+            options: Optional[dict] = None,
+            gcp_conn_id: str = 'google_cloud_default',
+            delegate_to: Optional[str] = None,
+            poll_sleep: int = 10,
+            job_class: Optional[str] = None,
+            check_if_running: CheckJobRunning.value = CheckJobRunning.WaitForRun,
```

Comment:
What is CheckJobRunning.value? Is it string int or something?

Reply:
It is `int`. I will put `int` here.

Reply:
it should be member of  value `CheckJobRunning` so it should be `CheckJobRunning` instead of `CheckJobRunning.value`